pkg/controllers/disruption/consolidation_test.go

```diff
@@ -214,6 +217,87 @@ var _ = Describe("Consolidation", func() {
 				metrics.ReasonLabel: "underutilized",
 			})
 		})
+		DescribeTable("should correctly report invalidated commands for emptiness disruption when", func(invalidatedReason string, validatorOpt TestEmptinessValidatorOption) {
```

Comment:
```suggestion
		DescribeTable("should correctly report invalidated commands for emptiness disruption", func(invalidatedReason string, validatorOpt TestEmptinessValidatorOption) {
```

nit: I'd include the "When" at the start of the entries